### PR TITLE
Add preload for css

### PR DESCRIPTION
**Pull request by JVolke**
### All changes meet the following requirements
- [ ] Changelog entry was added
- [x] Changes have been tested
- [x] Plugin can be built

@plentymarkets/ceres-io 
Added Preload for CSS and make the css easier to exchange.
The preload will improve the performance for Ceres slightly, especially for Google Pagespeed.

### CODE REVIEW

File: resources/views/PageDesign/Partials/Head.twig [hunk 1]
```diff
@@ -4,6 +4,7 @@
 <meta name="generator" content="plentymarkets" />
 
 {% set lang = services.sessionStorage.getLang() %}
+{% set defaultCSS = plugin_path('Ceres') ~"/css/ceres-legacy.css" %}
```
Comment (Tim-M-S) [inline resources/views/PageDesign/Partials/Head.twig]: space nach ~

File: resources/views/PageDesign/Partials/Head.twig [hunk 2]
```diff
@@ -78,8 +79,10 @@
 {% set performanceLevel = config('Ceres.log.performance.level') %}
 {% set itemName = config('Ceres.item.name') %}
 
+<link rel="preload" as="style" href="{{ defaultCSS }}">
 {% if LayoutContainer.show("Ceres::Template.StyleOverwrite") | trim is empty %}
-    <link data-sass-root="{{ plugin_path('Ceres') }}" data-sass-original="{{ plugin_path('Ceres') }}/css/ceres-legacy.scss" rel="stylesheet" href="{{ plugin_path('Ceres') }}/css/ceres-legacy.css">
+    <link data-sass-root="{{ plugin_path('Ceres')
```
Comment (Tim-M-S) [inline resources/views/PageDesign/Partials/Head.twig]: Bitte in einer Zeile

File: resources/views/PageDesign/Partials/Head.twig (hunk 1 above)
Comment (JVolke) [reply resources/views/PageDesign/Partials/Head.twig]: done

File: resources/views/PageDesign/Partials/Head.twig (hunk 2 above)
Comment (JVolke) [reply resources/views/PageDesign/Partials/Head.twig]: Ups, das war noch von einem anderen Test

**Review by Tim-M-S — APPROVED** [review]

**Comment by Tim-M-S** [comment]
Danke für Hilfe! @JVolke

**Pull request merged**